features/b.yml

```
@@ -0,0 +1,7 @@
+name: Bold element
+description: The `<b>` element is used to draw the reader's attention to the element's contents.
```

I don't think we should be shy about describing the feature in the way it's easiest to understand: it makes text bold. The definitions of some elements were contorted in the HTML5 area to not have anything purely presentational, but those semantics are quite abstract. I'd go with:

```suggestion
description: The `<b>` element makes text bold in order to draw the reader's attention to it.
```

And for the name, just "&lt;b>".

---

Haha, I haven't used or read about these elements in a very long time. Reading the MDN docs for `b`, `i`, and `u` shows how vague they now are: 

* "Bring Attention"
* "Idiomatic Text"
* "Unarticulated Annotation"

`s` is much clearer: "Strikethrough".

I think we should group them under just one feature only, and have a note in the `description` field about how they once were for text styling, but that CSS ways of achieving the same styles should now be used, and that semantic elements such as `strong` or `em` should be considered too.

I'm OK with being clear about the fact that, e.g., `b` makes the text bold, because that's what it's always done, and I don't see a user agent ever deviating from this. But I do think it's important for us to also say something about there being other, more current ways, of doing the same.

---

@foolip and I talked about the broader question earlier today: when should individual elements get their own feature? Are we really going to have individual features for these long-established things?

I think, in the near term, I favor individual features as a matter convenience until we recognize groups. I suspect we'll actually find our way to composite features (https://github.com/web-platform-dx/web-features/issues/971). So these elements may eventually find their way into something like "text annotation elements."